wiki/_Sidebar.md

```diff
@@ -0,0 +1,10 @@
+- [Installation](./Installation)
+- [Configuration](./Configuration)
+- [Usage](./Usage)
+- [Output](./Output)
+- [Plugins](./Plugins)
+- [Development](./Development%20-%20Index)
+  - [Overview](./Development%20-%20Overview)
+  - [Skeleton](./Development%20-%20Skeleton)
+  - [Setup](./Development%20-%20Setup)
+  - [Testing](./Development%20-%20Testing)
```

Comment:
Annoyingly GitHub wikis don't support subpaths so we need implement a custom sidebar. The `development` path is flattened so these links will work in a wiki setting 👍